pkg/controller/master/upgrade/version_syncer.go

```diff
@@ -174,21 +178,24 @@ func (s *versionSyncer) syncVersions(resp CheckUpgradeResponse, currentVersion s
 			}
 		}
 	}
+
 	return nil
 }
 
 // cleanupVersions remove version resources that's can't be upgraded to anymore
-func (s *versionSyncer) cleanupVersions(currentVersion string) error {
+func (s *versionSyncer) cleanupVersions(currentVersion string, Versions []Version) error {
```

Comment:
```suggestion
func (s *versionSyncer) cleanupVersions(currentVersion string, remoteVersions []Version) error {
```

Can we rename the variable to something like this for readability?